core/src/local_vote_signer_service.rs

```diff
@@ -15,10 +15,9 @@ pub struct LocalVoteSignerService {
 impl LocalVoteSignerService {
     #[allow(clippy::new_ret_no_self)]
     pub fn new(port_range: PortRange) -> (Self, SocketAddr) {
-        let addr = match solana_net_utils::find_available_port_in_range(port_range) {
-            Ok(port) => SocketAddr::new(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)), port),
-            Err(_e) => panic!("Failed to find an available port for local vote signer service"),
```

Comment:
This causes: 

```
error: `Err(_e)` matches all errors
  --> core/src/local_vote_signer_service.rs:20:13
   |
20 |             Err(_e) => panic!("Failed to find an available port for local vote signer service"),
   |             ^^^^^^^
   |
   = note: `-D clippy::match-wild-err-arm` implied by `-D warnings`
   = note: match each error separately or use the error output
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#match_wild_err_arm
```

Not suggested, but I think using map-expect is more better suited; it also prints the content of  `_e` automatically.